### PR TITLE
Add description label to Dockerfile

### PR DESCRIPTION
**Pull request by amitkgupta**
Some image scanning tools that check for vulnerabilities and OCI image best
practices expect to find a `description` label on images that is not inherited
from base images. The proposed change addresses this requirement.

### CODE REVIEW

**Review by jessemillerjom — APPROVED** [review]
lgtm

**Comment by amitkgupta** [comment]
Hey @jessemillerjom what's the process as it relates to merging PRs for this repo? Should I merge once there's a successful review, or should I wait for one of the owners/maintainers to merge? If I should merge, do you have a preference of Create a merge commit, Squash and merge, or Rebase and merge?

**Comment by srpanwar-confluent** [comment]
> Hey @jessemillerjom what's the process as it relates to merging PRs for this repo? Should I merge once there's a successful review, or should I wait for one of the owners/maintainers to merge? If I should merge, do you have a preference of Create a merge commit, Squash and merge, or Rebase and merge?

@amitkgupta Thanks for the change. I can merge it for you

**Comment by srpanwar-confluent** [comment]
@amitkgupta forgot to ask. Do we want to back port this change?

**Comment by amitkgupta** [comment]
Hey @srpanwar-confluent I PR'd this to 6.2.x and master. It would be pretty reasonable to backport it and forwardport(?) it to 7.0.x as well. Is there an easy way to do that?

**Comment by srpanwar-confluent** [comment]
forward port works better. That is how we do for most project except ce-kafka. You might want to cherry-pick the change 6.2.x based branch

**Pull request closed without merging**
